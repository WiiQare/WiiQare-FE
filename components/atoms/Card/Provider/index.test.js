import "@testing-library/jest-dom";
import { render, screen } from "@testing-library/react";
import CardProvider from ".";

describe("CardProvider", () => {
  test("should render CardProvider component", () => {
    const { container } = render(<CardProvider packages={[]} />);
    expect(container).toMatchSnapshot();
  });
});
