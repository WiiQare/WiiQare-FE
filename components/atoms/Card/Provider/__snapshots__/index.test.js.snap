// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardProvider should render CardProvider component 1`] = `
<div>
  <div
    class="border hover:shadow rounded-lg py-4 px-5 flex flex-col md:flex-row justify-between gap-6 bg-white cursor-pointer"
  >
    <div
      class="flex gap-3"
    >
      <div
        class="w-32 h-24 md:w-40 md:h-32 border py-2 rounded-lg overflow-hidden"
      >
        <img
          alt="undefined Logo"
          class="object-contain w-full h-full"
          data-nimg="1"
          decoding="async"
          loading="lazy"
          src=""
          style="color: transparent;"
        />
      </div>
      <div
        class="flex flex-col gap-3 md:w-72"
      >
        <div
          class="text-gray-700 space-y-2"
        >
          <h3
            class="text-lg md:text-lg font-bold"
          />
          <span
            class="text-sm font-light"
          />
        </div>
        <span
          class="flex gap-3 items-center text-sm text-gray-700 font-light"
        >
          <span
            class="bg-orange px-2 py-1 text-white text-sm rounded-md w-fit flex gap-3 items-center"
          >
            0
             Plan
          </span>
        </span>
      </div>
    </div>
    <div
      class="text-gray-700 block items-center gap-4 md:space-y-2 md:w-64"
    >
      <h5
        class="font-light"
      >
        Adresse
      </h5>
      <h3
        class="text-2xl font-semibold"
      >
        <span
          class="text-sm"
        />
      </h3>
    </div>
    <div
      class="self-center flex md:flex-col gap-3"
    >
      <button
        class="bg-blue-600 text-white font-extralight py-3 md:px-9 px-5 rounded-lg text-sm"
      >
        En savoir plus
      </button>
    </div>
  </div>
</div>
`;
